lib/Frontend/CompilerInvocation.cpp

```diff
@@ -2101,7 +2145,7 @@ static bool ParseSILArgs(SILOptions &Opts, ArgList &Args,
                          const FrontendOptions &FEOpts,
                          const TypeCheckerOptions &TCOpts,
                          DiagnosticEngine &Diags,
-                         const llvm::Triple &Triple,
+                         LangOptions &LangOpts,
```

Comment:
Do I understand correctly that moving the option from Frontend to Lang was to be able to use the Features flags? I think I've hit that restriction before myself.

Reply:
Yes unfortunately.